shell.nix

```diff
@@ -8,6 +8,9 @@ pkgs.mkShell {
     nettools
     niv
     gmp
+    docker-compose
+    nixfmt
+    colima
```

Comment:
This is a way to use `docker` on MacOS without having to install Docker Desktop which is not free anymore in some cases.